octobot_trading/exchanges/exchange_manager.py

```diff
@@ -294,15 +292,34 @@ async def _create_exchange_producers(self):
             for updater in AUTHENTICATED_UPDATER_SIMULATOR_PRODUCERS:
                 await updater(get_chan(updater.CHANNEL_NAME, self.id)).run()
 
+    async def _create_authenticated_producers(self):
+        for updater in AUTHENTICATED_UPDATER_PRODUCERS:
+            if self._is_managed_by_websocket(updater.CHANNEL_NAME):
+                # websocket is handling this channel: initialize data if required
+                if self._is_websocket_feed_requiring_init(updater.CHANNEL_NAME):
+                    try:
+                        create_task(updater(get_chan(updater.CHANNEL_NAME, self.id)).fetch_and_push())
```

Comment:
I'm not very fond of creating the task here. I don't know where to move this `create_task` but I think this is not a good place for it.

Reply:
For now ExchangeManager doesn't manage tasks, I don't think it should

Reply:
Agree, what about putting it in octobot_websocket ? This class is already handling the websocket pool executor

Reply:
Hum, I don't like too. In my opinion, updaters should manage their own tasks. We should probably create this task in `fetch_and_push` or call another method that will create the task for `fetch_and_push`.

Reply:
fetch_and_push should not create its own task in my opinion. Where do you see this method creating a task ?

Reply:
No I agree. A method in updater should do the job of creating a task for `fetch_and_push`.

Reply:
How do we call it ? `trigger_single_update()` ?